helpers.go

```diff
@@ -70,3 +73,118 @@ func getListOfVersionedRepositories(inVersion string) ([]string, error) {
 
 	return strings.Split(strings.TrimSpace(string(output)), "\n"), nil
 }
+
+func validDependabotPR(pr *github.PullRequestEvent) bool {
+	// NOTE: For now, checking the 'labeled' condition, and not 'opened' (for testing purposes)
```

Comment:
Unsure, whether this is the right check, once this is live

Reply:
I don't think this will be enough, as the bot might add different labels. I think it is safer with the "open" condition, but maybe you had a good reason for not using it?

Reply:
Yes, testing locally with the request you sent me :P 
But for the final, opened should be it. Just, could you please double check that this shows up in of the the `webhook` requests?

Reply:
Sorry. I sent you again a new one, this one should be correct.